juniper_official/routing/check-ospf-statistics.rule

```diff
@@ -79,14 +77,16 @@
                     }
                 }
                 term is-ospf-hello-receive-not-increasing {
+                    when {
+                        matches-with-previous "$hello-received-5sec";
```

Comment:
We should probably do red first and then green for these cases as present logic has some scenarios uncovered
let say threshold is 2 previous value is 10 and present is 11. In this case we will not write anything to trigger table